src/pages/Search/SearchArea/SearchArea.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
Her gir det mest mening å teste integrasjonen mellom search-komponentene på samme sted, altså i Search.test.tsx. Dersom du flytter testene i SearchArea.test.tsx og SearchResults.test.tsx dit, kan vi teste hele brukerflyten på ett sted, og samtidig oppnå akkurat samme testdekning. I tillegg sparer vi oss fra å skrive mye unødvendig oppsett-kode. (For eksempel: A: bruker velger søk. B: Bruker skriver noe. C: Bruker ser søkeresultater. D: Bruker velger et annet søk (og søkefeltet endrer seg). E: Bruker trykker på samme knapp igjen, og "saved searches" viser). 

Kilder:
Prioriter integrasjonstester over unit-tester: https://kentcdodds.com/blog/write-tests

Bakgrunn: Bør jeg skrive test for parent- eller child komponenten? https://www.youtube.com/watch?v=0qmPdcV-rN8&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u